x/wasm/keeper/relay.go

```diff
@@ -60,8 +76,20 @@ func (k Keeper) OnConnectChannel(
 	env := types.NewEnv(ctx, contractAddr)
 	querier := k.newQueryHandler(ctx, contractAddr)
 
+	msg := wasmvmtypes.IBCChannelConnectMsg{}
+	if counterpartyVersion == "" {
```

Comment:
I don't really know this codebase, but would it be possible to rely on something more meaningful for this?
```go
if msgConfirm {
    // ..
}
```

Reply:
Good question, captured in https://github.com/CosmWasm/wasmd/issues/561